client/src/components/payments/stripeStyles.js

```diff
@@ -0,0 +1,42 @@
+//stripe elements span card info, form, inputs, button
+//of payment card. If you figure out how to use MUI
+//components for these items please let me know. -taryn
+
+const buttonStyle = {
+    width: '220px',
+    height: '64px',
+    color: '#ffffff',
+    backgroundColor: true ? '#69E781' : 'rgb(73, 161, 90)',
+    fontSize: '1em',
+    textTransform: 'uppercase',
+    '&:hover': {
+        color: 'purple'//not working
```

Comment:
I know this is not working, but shouldn't you use the theme primary color or shades of it?